frontend/src/pages/IstioConfigNew/GatewayForm/ListenerList.tsx

```diff
@@ -108,7 +114,8 @@ export const ListenerList: React.FC<ListenerListProps> = (props: ListenerListPro
       port: 70000,
       name: '',
       protocol: protocols[0],
-      allowedRoutes: { namespaces: { from: allowedRoutes[0], selector: { matchLabels: {} } } }
+      allowedRoutes: { namespaces: { from: allowedRoutes[0], selector: { matchLabels: {} } } },
+      tls: null
```

Comment:
```suggestion
      allowedRoutes: { namespaces: { from: allowedRoutes[0] } }
```
Remove empty values by default